### DISCUSSION TITLE
Should we exclude PRs from dependabot and renovate bots from the "immediate response" workflow?

### DISCUSSION THREAD

**Discussion by G-Rath** [octokit/.github#56] [General]
I've implemented this here https://github.com/octokit/webhooks/commit/097fd535f41aa27b0d0dad325d398000e8213582 but since this workflow exists in multiple repos making a discussion for it first felt right.

It's a small thing, but it would reduce the noise notification wise

**Comment by oscard0m** [comment]
IMO yes

**Comment by gr2m** [comment]
I vote yes, too.

We could even consider to only post the message to pull requests coming from forks. All others are contributors and know

**Reply by G-Rath** [reply]
Do you know if the GitHub Actions event context provides that information? either way we can start with the bots and hopefully add that later if we find a way

**Reply by gr2m** [reply]
we can check if 

1. the event is not "pull_request"
2. or repository owner is not `octokit`

```yml
    if: ${{ github.event_name != 'pull_request' && github.repository_owner != 'octokit' }}
```

THat should be true when triggered by

1. The `issues` event
2. The `pull_request` event when coming from a fork

**Comment by G-Rath** [comment]
I don't know if we've got some tool for bulk updating workflows across repos so I've just opened https://github.com/octokit/webhooks/pull/840 to start with

**Reply by wolfy1339** [reply]
You can write a simple octoherd script to do the modifications

**Reply by G-Rath** [reply]
sweet I'll look into that at some point - for now I'm just going to manually do a few of the big ones

**Comment by wolfy1339** [comment]
This has been resolved as far as renovate and dependabot go.

We should probably also exclude octokitbot, and githubactions[bot] as well
